pkgs/development/tools/repository-managers/nexus/default.nix

```diff
@@ -1,29 +1,48 @@
-{ stdenv, fetchurl, makeWrapper, jre }:
+{ stdenv, fetchurl, makeWrapper, jre, gawk }:
 stdenv.mkDerivation rec {
   name = "nexus-${version}";
-  version = "2.12.0-01";
+  version = "3.5.1-02";
 
   src = fetchurl {
-    url = "https://sonatype-download.global.ssl.fastly.net/nexus/oss/nexus-${version}-bundle.tar.gz";
-    sha256 = "1k3z7kwcmr1pxaxfnak99fq5s8br9zbqbfpyw1afi86ykkph4g5z";
+    url = "https://sonatype-download.global.ssl.fastly.net/nexus/3/nexus-${version}-mac.tgz";
```

Comment:
The macOS bundle works cross-platform?!

Reply:
Yes because there are only two diffrents:
1. in .install4j folder there comes also a jre.bundle in mac style (if no local jre is installed)
2. Additional checks for jre in mac style (include fallback to jre.bundle) in startup shell script

And both has no side effects on unix systems (expect the extra 167M for the jre bundle)

I test it on my nixos box.